### DISCUSSION TITLE
MIDI Clock Generator

### DISCUSSION THREAD

**Discussion by gh0stless** [shanteacontrols/OpenDeck#151] [Ideas]
I want to realize my dream controller for Rebirth RB-338.
I want to add a MIDI clock generator to the firmware (OpenDeck Hardware 1).
Do you think there are reasons against it?

**Comment by paradajz** [comment]
I'm not sure what are you asking here.

**Comment by gh0stless** [comment]
I mean: That the OpenDeck controller should be able to generate MIDI clock messages: With changeable speed (assign a knob or encoder).

**Reply by paradajz** [reply]
I see. It's possible, yes, but not on version 1 of the board.

**Comment by paradajz** [comment]
MIDI clock is available as an official feature since v7.6.0 firmware.
